scripts/dp_normalize.py

```diff
@@ -0,0 +1,133 @@
+import logging
+from pathlib import Path
+import yaml
+
+import pandas as pd
+from pycytominer import normalize
+
+
+def infer_dp_features(dp_profile) -> list[str]:
```

Comment:
it's really not ideal to have this function (and a couple others) to be defined in 3 different places! 🙃 

if you add a `__init__.py` file in the folder the function is initially defined, can it be imported? i.e. a function can be imported from a different location even if the package is not pip installed. (but these imports can get tricky)

Reply:
Yeah, this is were the redundant error starts to be a pain. Using `__init__.py` will not work because the execution of this scripts are encapsulated with the environment files we use. 

Since the environment files does not contain `cytosnake` as a dependencies, it has not access to all its code. :( So even importing the main module `cytosnake` will not be recognized. 

Hope this helps.

Reply:
https://snakemake.readthedocs.io/en/stable/project_info/faq.html#i-want-to-import-some-helper-functions-from-another-python-file-is-that-possible

> [I want to import some helper functions from another python file. Is that possible?](https://snakemake.readthedocs.io/en/stable/project_info/faq.html#id24)

> Yes, from version 2.4.8 on, Snakemake allows to import python modules (and also simple python files) from the same directory where the Snakefile resides.

Are we using 2.4.8? Looks like we can import _from the same directory where the Snakefile resides_. Is this impossible for us? I don't like the precedent of duplicate code here, but if it's impossible then... I guess we have no other option!

Reply:
We are using `7.8.0+`. This is true, we can import functions from other python helper modules. However, since these rules are encapsulated in `conda` environments, those modules do not exists due to `cytosnake` not being a requirement within the `yaml` files.